packages/react-scripts/scripts/build.js

```diff
@@ -40,7 +40,36 @@ const configFactory = require('../config/webpack.config');
 const paths = require('../config/paths');
 const checkRequiredFiles = require('react-dev-utils/checkRequiredFiles');
 const formatWebpackMessages = require('react-dev-utils/formatWebpackMessages');
-const printHostingInstructions = require('react-dev-utils/printHostingInstructions');
+
+//
+// Override the default hosting instructions with FTW specific instructions.
```

Comment:
```suggestion
// Sharetribe custom: override the default hosting instructions with FTW specific instructions.
```

We have used this string ("Sharetribe custom:") in all the places that has been customized from default CRA project.